### PR TITLE
Infinite merge

### PR DESCRIPTION
**Pull request by c-schwarze**

### CODE REVIEW

File: app/static/src/js/ajaxCall.js [hunk 1]
```diff
@@ -2,50 +2,82 @@ const results = document.querySelector('#results')
...
-                console.log('loading');
-            } else if (xhr.status >= 500 || xhr.status == 0) {
+    obj = {};
+    [...fd.entries()].forEach(entry => obj[entry[0]] = entry[1])
+
+    function postAjax (url, data, callback) {
+        const params = typeof data === 'string' ? data : Object.keys(data).map(
+            function (k) { return encodeURIComponent(k) + '=' + encodeURIComponent(data[k]) }
+        ).join('&')
+        console.log(params)
```
Comment (phg49389) [inline app/static/src/js/ajaxCall.js]: Do we want this `console.log` to stay in the code?

File: app/static/src/js/ajaxCall.js [hunk 2]
```diff
@@ -2,50 +2,82 @@ const results = document.querySelector('#results')
...
+                const new_page_number = parseInt(infiniteDataContainer.getAttribute('page')) + 1
+                infiniteDataContainer.setAttribute('page', new_page_number)
+
+                // get data and append page number to the data
+                obj = infiniteDataContainer.getAttribute('data') + '&page=' + new_page_number;
+                postAjax('/search', obj, function(xhr){
+                    // append results to the other results
+                    infiniteScroll.innerHTML = infiniteScroll.innerHTML + xhr.responseText;
+
+                    // allow the
```
Comment (phg49389) [inline app/static/src/js/ajaxCall.js]: This comment seems unfinished

Comment (c-schwarze) [reply app/static/src/js/ajaxCall.js]: heh. Yeah there are a few things that were "fixed" but it had issues on the merge an they got lost

File: app/static/src/js/ajaxCall.js (hunk 1 above)
Comment (c-schwarze) [reply app/static/src/js/ajaxCall.js]: nope. done

**Pull request merged**